packages/uppy/types/index.test-d.ts

```diff
@@ -1,7 +1,7 @@
 import * as Uppy from '..';
 
 (() => {
-  const uppy = new Uppy.Core({ autoProceed: false })
+  const uppy = new Uppy.Uppy({ autoProceed: false })
```

Comment:
Should we keep `Uppy.Core`, among with `Uppy.Uppy` in the bundle? Could release it as a patch.

Reply:
I’d be up for at least export a function named `Core` that throws `Please use Uppy instead of Core`, to improve the migration workflow.